src/chainparams.cpp

```diff
@@ -795,6 +798,7 @@ class CRegTestParams : public CChainParams {
         consensus.DIP0003EnforcementHeight = 500;
         consensus.DIP0003EnforcementHash = uint256();
         consensus.DIP0008Height = 432;
+        consensus.MinBIP9WarningHeight = 0;
```

Comment:
16713: should probably be 432, since we do not have dip8 "always active" on regtest unlike segwit in the original PR

Reply:
This breaks the `feature_versionbits_warning.py` functional test and has been reverted!

Error Log:
```
2021-12-12T15:00:08.767000Z TestFramework (INFO): Initializing test directory /tmp/dash_func_test_nlm9f64q
2021-12-12T15:00:15.085000Z TestFramework (INFO): Check that there is no warning if previous VB_BLOCKS have <VB_THRESHOLD blocks with unknown versionbits version.
2021-12-12T15:00:15.532000Z TestFramework (INFO): Check that there is a warning if previous VB_BLOCKS have >=VB_THRESHOLD blocks with unknown versionbits version.
2021-12-12T15:00:18.179000Z TestFramework (ERROR): Assertion failed
Traceback (most recent call last):
  File "/dash-src/test/functional/test_framework/test_framework.py", line 233, in main
    self.run_test()
  File "./test/functional/feature_versionbits_warning.py", line 98, in run_test
    assert WARN_UNKNOWN_RULES_ACTIVE in node.getmininginfo()["warnings"]
AssertionError
2021-12-12T15:00:18.230000Z TestFramework (INFO): Stopping nodes
2021-12-12T15:00:19.183000Z TestFramework (WARNING): Not cleaning up dir /tmp/dash_func_test_nlm9f64q
2021-12-12T15:00:19.183000Z TestFramework (ERROR): Test failed. Test logging available at /tmp/dash_func_test_nlm9f64q/test_framework.log
2021-12-12T15:00:19.184000Z TestFramework (ERROR): Hint: Call /dash-src/test/functional/combine_logs.py '/tmp/dash_func_test_nlm9f64q' to consolidate all logs
```